### ISSUE TITLE
Cursor is kept hidden after opening action menu with 'set termguicolors'

### ISSUE THREAD

**Issue by ChrisAmelia** [iamcco/coc-actions#12]
init.nvim
```vim
set runtimepath^=~/.local/share/nvim/plugged/coc.nvim
filetype plugin indent on
syntax on
set hidden
set termguicolors
```

coc-settings.json
```json
"coc-actions.hideCursor": true
```

To reproduce: open any file then call `:CocCommand actions.open`.

**Comment by iamcco** [comment]
Close, it should be fixed according to #16. Feel free to reopen if the issue still exists.

**Comment by ChrisAmelia** [comment]
Updated to version 1.4.1.
Tested with above config, the issue still exists.

A workaround is to set `"coc-actions.hideCursor"` to false.
